soccer/Processor.hpp

```diff
@@ -93,16 +93,18 @@ class Processor : public QThread {
     }
 
     void manualID(int value);
-    int manualID() const { return _manualID; }
+    int manualID() const { return _context.game_settings.manualID; }
```

Comment:
You should be able to delete these, right?

Either way, you should _definitely_ get rid of the setters' declarations in here.